### PR TITLE
Fix package_dir to properly reference lib. Remove extraneous include.

### LINKED ISSUE #558: Can't install directly from github using pip -e
I wanted to install ansible directly from github, using pip -e. Unfortunately, it fails to install properly:

```
$ pip install -e git://github.com/ansible/ansible.git#egg=ansible
Obtaining ansible from git+git://github.com/ansible/ansible.git#egg=ansible
  Cloning git://github.com/ansible/ansible.git to ./.virtualenvs/ansible/src/ansible
  Running setup.py egg_info for package ansible

    warning: no files found matching 'packaging/distutils/setup.py'
Requirement already satisfied (use --upgrade to upgrade): paramiko in ./.virtualenvs/ansible/lib/python2.7/site-packages (from ansible)
Requirement already satisfied (use --upgrade to upgrade): jinja2 in ./.virtualenvs/ansible/lib/python2.7/site-packages (from ansible)
Requirement already satisfied (use --upgrade to upgrade): PyYAML in ./.virtualenvs/ansible/lib/python2.7/site-packages (from ansible)
Requirement already satisfied (use --upgrade to upgrade): pycrypto>=2.1 in ./.virtualenvs/ansible/lib/python2.7/site-packages (from paramiko->ansible)
Installing collected packages: ansible
  Running setup.py develop for ansible

    warning: no files found matching 'packaging/distutils/setup.py'
    Creating /Users/lorin/.virtualenvs/ansible/lib/python2.7/site-packages/ansible.egg-link (link to .)
    Adding ansible 0.6 to easy-install.pth file
    Installing ansible script to /Users/lorin/.virtualenvs/ansible/bin
    Installing ansible-playbook script to /Users/lorin/.virtualenvs/ansible/bin

    Installed /Users/lorin/.virtualenvs/ansible/src/ansible
Successfully installed ansible
```

When I try to run, it fails:

```
$ ansible
Traceback (most recent call last):
  File "/Users/lorin/.virtualenvs/ansible/bin/ansible", line 7, in <module>
    execfile(__file__)
  File "/Users/lorin/.virtualenvs/ansible/src/ansible/bin/ansible", line 25, in <module>
    from ansible.runner import Runner
ImportError: No module named ansible.runner
```

Installing version 0.5 from the tarball works fine:

```
pip install https://github.com/downloads/ansible/ansible/ansible-0.5.tar.gz
```

### PR DESCRIPTION
**Pull request by jsdalton**
This pull request fixes some of the package details in setup.py and allows for ansible to be installed in develop mode (a.k.a. `pip install -e https://github.com/ansible/ansible.git`).

It fixes #558 (which was closed pending a proposed fix, which this pull request is).

The problem boiled down to a slightly incorrect construction of the `package_dir` argument to `setup()`.

Proper usage of this setting is described [here in the disutils docs](http://docs.python.org/2/distutils/setupscript.html#listing-whole-packages). Those docs actually perfectly describe the ansible set up, where source code lives under a `lib` subdirectory. They suggest using `{ '': 'lib' }` and then explicitly listing out the packages that live there. This is what my pull request accomplishes.

That's the easy explanation, and if you read those docs I linked to above my change aligns with the recommendation there.

The more confusing bit is the fact that this _should_ apparently play nicely with `pip install -e` / `python setup.py develop` as it is written now. Why it doesn't play nicely is apparently an issue that the distribute folks are contending with, see here: https://bitbucket.org/tarek/distribute/issue/177/setuppy-develop-doesnt-support-package_dir

In short, this change should not alter behavior for normal `python setup.py install` usage but should allow developers to install in editable mode via pip if they choose to.

(The change to MANIFEST.in removes an include that references a non existent file and throws a warning during pip install process.)

### CODE REVIEW

**Comment by jsdalton** [comment]
Sadly, there is more install magic happening under the covers than I realized and this isn't sufficient to allow for `python setup.py develop` (although it at least gets ansible on the sys path). I'll hack at it later and re-submit if it turns out to be worth contributing.
